soc/openhwgroup/cva6/cv32a6/Kconfig.soc

```diff
@@ -0,0 +1,21 @@
+#Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+#SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_CV32A6
+	bool
+	select SOC_FAMILY_OPENHWGROUP_CVA6
+	help
+	  CVA6 32-bit RISC-V CPUs, commonly used as soft cores on FPGAs.
+	  This configuration works with the upstream project (https://github.com/openhwgroup/cva6).
+
+config SOC_CV32A6
+	bool
+	select SOC_SERIES_CV32A6
+	help
+	  Minimal CVA6 core in 32-bit configuration: ima extensions, no MMU or PMP,
+	  SoC with CLINT, PLIC, UART, SPI, and Ethernet (Xilinx or lowRISC).
+config SOC_SERIES
+	default "cv32a6" if SOC_SERIES_CV32A6
+
+config SOC
+	default "cv32a6" if SOC_CV32A6
```

Comment:
There are several slightly different versions of the CVA6 32-bit SOC: with and without MMU, with and without FPU, and a RV32E version is currently being developed. I have added the least common denominator here, future PRs can add the other variants as soon as they are finalized.

Reply:
All of the RV32I SoCs work with the config I provided, but the RV32E really needs to be separate (it has fewer registers).

Reply:
Are they going to be under the CV32A6 soc series or something else? Because if it's going to be just a 1:1 mapping of SoC to SoC series, the series can go, it's not required

Reply:
At least the embedded version will be - it has a near-identical configuration, but with the RV32E instead of the RV32I instruction set. There is also an ASIC version of the 32-bit SoC (the CV32A65X by Thales), which has some additional instruction set extensions enabled.